app/questionnaire/value_source_resolver.py

```diff
@@ -163,12 +164,19 @@ def _resolve_list_value_source(self, value_source: Mapping) -> int | str | list:
 
     def _resolve_calculated_summary_value_source(
         self, value_source: Mapping, *, assess_routing_path: bool
-    ) -> IntOrDecimal:
+    ) -> IntOrDecimal | None:
         """Calculates the value for the 'calculation' used by the provided Calculated Summary.
 
-        The caller is responsible for ensuring the provided Calculated Summary and its answers are on the path.
+        The caller is responsible for ensuring the provided Calculated Summary and its answers are on the path,
```

Comment:
This comment is no longer true?

Reply:
is `self.routing_path_block_ids` is None, (as in quite a few tests) then it will return the sum of the calculated summary regardless of whether it is on the path